vba_timeseries/models/exp/m_suuvid_obs_exp.m

```diff
@@ -0,0 +1,48 @@
+function  [ gx ] = m_suuvid_obs_exp(Xt, phi, u, inG)
+% INPUT
+% - x_t : hidden states (weights of basis functions)
+% - phi : temperature (1x1)
+% - u   : imput vector (not used in observation)
+% - inG : multinomial
+% OUTPUT
+% - gx : p(chosen|x_t)
+
+phi = transform_phi(phi, inG);
+gamma = phi(1); %slope on logistic (sensitivity)
+nu = phi(2); %basal vigor as a function of time elapsed
+kappa = phi(3); %softmax temperature
+omega = phi(4); %stickiness
+
+tdiff = u(4); %cross-check position in u
+active_action = u(5); %cross-check position in u
+
+n_actions = inG.hidden_states;
+
+Qcur = Xt(1:n_actions);
+Qtot = sum(Qcur); %total value
+
+%probability of responding at all: pure 2PL
+%tdiffR = tdiff/1000;
+%if tdiffR>1
+%        disp('tdiff reset')
+%        tdiffR = 1;
+%end
+%p_respond = 1/(1 + exp(-gamma * Qtot * (tdiffR - nu)));
+%p_respond = 1/(1 + exp(-gamma * Qtot * (tdiff/1000 + nu)));
+p_respond = 1-expexp(-1*(nu + gamma*Qtot)*tdiff/1000);
```

Comment:
I think you meant just 'exp', not 'expexp'. I would also delete the two commented p_respond lines above to avoid confusion with past models.

Reply:
Great catch! Really appreciate it!